albumentations/augmentations/domain_adaptation.py

```diff
@@ -200,3 +220,85 @@ def get_transform_init_args_names(self):
 
     def _to_dict(self):
         raise NotImplementedError("FDA can not be serialized.")
+
+
+class PixelDistributionAdaptation(ImageOnlyTransform):
+    """
+    Another naive and quick pixel-level domain adaptation. It fits a simple transform (such as PCA, StandardScaler
+    or MinMaxScaler) on both original and reference image, transforms original image with transform trained on this
+    image and then performs inverse transformation using transform fitted on reference image.
+
+    Args:
+        reference_images (List[str] or List(np.ndarray)): List of file paths for reference images
+            or list of reference images.
+        blend_ratio (float, float): Tuple of min and max blend ratio. Matched image will be blended with original
+            with random blend factor for increased diversity of generated images.
+        read_fn (Callable): Used-defined function to read image. Function should get image path and return numpy
+            array of image pixels.
+        p (float): probability of applying the transform. Default: 1.0.
+        transform_type (str): type of transform; "pca", "standard", "minmax" are allowed.
+
+    Targets:
+        image
+
+    Image types:
+        uint8, float32
+
+    See also: https://github.com/arsenyinfo/qudida
+    """
+
+    def __init__(
+        self,
+        reference_images: List[Union[str, np.ndarray]],
+        blend_ratio: Tuple[float, float] = (0.25, 1.0),
+        read_fn: Callable[[Union[str, np.ndarray]], np.ndarray] = read_rgb_image,
+        always_apply=False,
+        p=0.5,
+        transform_type: str = "pca",
+    ):
+        super().__init__(always_apply=always_apply, p=p)
+        self.reference_images = reference_images
+        self.read_fn = read_fn
+        self.blend_ratio = blend_ratio
+        assert transform_type in ("pca", "standard", "minmax")
+        self.transform_type = transform_type
+
+    @staticmethod
+    def _validate_shape(img: np.ndarray):
+        if len(img.shape) != 3:
```

Comment:
To check that image is gray we have function [is_grayscale_image](https://github.com/albumentations-team/albumentations/blob/95a048d342cbe748e5acf15cb1a28611a6303885/albumentations/augmentations/functional.py#L82)